### ISSUE TITLE
In "REVIEW.." view, Subordinates are confusingly followed by "Units: 0"

### ISSUE THREAD

**Issue by battlemidget** [conjure-up/conjure-up#603]
When you have a subordinate (like flannel in k8s core) displayed in the "REVIEW AND CONFIGURE APPLICATIONS" screen, it indicates "Units: 0". That's kind of untrue, and left me wondering if I was going to get flannel or not.

**Comment by mikemccracken** [comment]
Agreed, this is misleading. I think we should just not show the "Units" text for subs.

**Comment by johnsca** [comment]
+1 on not showing Units box for subordinates, but we should also figure out how many instances of the subordinates we expect so that we can have the Deploy Status screen be correct from the get-go.
